_world_speakers/ryan-singer.md

```diff
@@ -0,0 +1,13 @@
+---
+first_name: Ryan
+last_name: Singer
+image_path: /assets/world/images/speakers/r-singer.jpg
+role:
```

Comment:
@ayushn21 is this intentionally empty? 
It generates a weird look with the comma on the page:

![image](https://github.com/rails/website/assets/752058/79e7f726-d339-40ac-8504-876466823f27)